browser/ui/webui/ai_chat/ai_chat_ui_page_handler.cc

```diff
@@ -122,6 +242,11 @@ void AIChatUIPageHandler::SubmitHumanConversationEntry(
   DCHECK(!active_chat_tab_helper_->IsRequestInProgress())
       << "Should not be able to submit more"
       << "than a single human conversation turn at a time.";
+
+#if BUILDFLAG(ENABLE_PRINT_PREVIEW)
+  MaybeCreatePrintPreview();
```

Comment:
Not really, that is triggered on each message only for docs.google.com now. It will be refactored out on my next PR which will use print preview extraction on image based pdf.